### PR TITLE
todai.kawai-juku.ac.jp

### LINKED ISSUE #108195: todai.kawai-juku.ac.jp
### Issue URL (Social Widget)
 [https://todai.kawai-juku.ac.jp/joy/2021/list6.php](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftodai.kawai-juku.ac.jp%2Fjoy%2F2021%2Flist6.php)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.2
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese, <br/>AdGuard Chinese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #108195

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -116,6 +116,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+todai.kawai-juku.ac.jp###pushBnr
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/popups.txt:119]: Remove this and instead add `||pushcode.jp^$third-party` to push-notifications.txt, this is the cause.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -49,6 +49,7 @@ kriptoradar.com,morlevha.com,korsanedebiyat.com,otomobilgunluklerim.com,tilki.ne
...
 !
+todai.kawai-juku.ac.jp##.ShareBox
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:52]: Hmmmm, on my end it's `##.snsShareBox`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/popups.txt]: ok

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:52]: sorry,I missed

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**